src/ast/mod.rs

```diff
@@ -3401,6 +3401,13 @@ pub enum Statement {
         partitioned: Option<Vec<Expr>>,
         table_format: Option<HiveLoadDataFormat>,
     },
+    /// ```sql
+    /// Rename TABLE tbl_name TO new_tbl_name[, tbl_name2 TO new_tbl_name2] ...
+    /// ```
+    /// Renames one or more tables
+    ///
+    /// See Mysql <https://dev.mysql.com/doc/refman/9.1/en/rename-table.html>
+    RenameTable(Vec<RenameObjectDef>),
```

Comment:
```suggestion
    RenameTable(Vec<RenameTable>),
```
Could we call `RenameTable`? that would match the variant name, and the object def terminology I think would become too specific or no longer descriptive if other fields are added to the struct

Reply:
Done!